components/02-elements/rating/_rating.scss

```diff
@@ -1,78 +1,108 @@
-$rating-font-size: $font-size-large;
-$rating-active   : $orange;
+$rating__size: 96px;
+$rating__item-size: $rating__size / 5;
+$icon__active-color: '#fab216';
+$icon__active-color--with-border: '#fab216';
+$icon__inactive-color: '#c2c1df';
+
 
 .rating {
-    unicode-bidi: bidi-override;
-    direction: rtl;
-    font-size: $rating-font-size;
+    width: $rating__size;
     cursor: pointer;
-    text-align: left;
 
-    span {
-        display: inline-block;
+    &--rate {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        direction: rtl;
```

Comment:
you have two modifiers on main element: `--rate` and `--summary`, IMO, you should delete one and style it for default... I cannot see the `--summary` so maybe should be deleted from hbs template? But IMO, it's not quite logic that you have `summary` variant and in css `--rate` (and not `--summary` modifier). And do you think?

Reply:
yes. after refactoring changing variants is good idea. done